test/python/test_ibmqjob.py

```diff
@@ -250,6 +265,32 @@ def test_get_backend_name(self):
         job = backend.run(quantum_job)
         self.assertTrue(job.backend_name == backend_name)
 
+    def test_get_jobs_from_backend(self):
+        backends = self._provider.available_backends()
+        backend = lowest_pending_jobs(backends)
+        start_time = time.time()
+        job_list = backend.jobs(limit=5, skip=0)
+        self.log.info('time to get jobs: %0.3f s', time.time() - start_time)
+        self.log.info('found %s jobs on backend %s', len(job_list), backend.name)
+        for job in job_list:
+            self.log.info('status: %s', job.status)
+            self.assertTrue(isinstance(job.id, str))
+        self.log.info('time to get job statuses: %0.3f s', time.time() - start_time)
+
+    def test_retrieve_job(self):
```

Comment:
I would make a test for a Job with multiple circuits.

Reply:
test_run_simulator does this

Reply:
Ok!